### PR TITLE
Added endpoint to send change email requests. 

### PR DESCRIPTION
**Pull request by joseluisgraa**
Added endpoint to send change email requests. 
Setting uid, expiry and signature in window object, values sent back with requests to check if it is legitimate and not expired. 
Limiting number of requests with same uid.

### CODE REVIEW

File: src/app/b2c/postChangeEmail.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  if (!securedEndpointUrl) {
+    res.writeHead(404);
+    res.write("Change email endpoint not configured properly");
+    res.end();
+    return;
+  }
+
+  //check this uid hasn't been used too many times
+  if (countRequest(req.headers) > 5) {
+    res.writeHead(500);
```
Comment (DrogoNevets) [inline src/app/b2c/postChangeEmail.js:89]: prefer `res.status`

File: src/app/b2c/postChangeEmail.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    res.write("Change email endpoint called too many times");
```
Comment (DrogoNevets) [inline src/app/b2c/postChangeEmail.js:90]: prefer `res.send(...)`

File: src/app/b2c/postChangeEmail.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    res.end();
```
Comment (DrogoNevets) [inline src/app/b2c/postChangeEmail.js:91]: can be chained
`res.status(500).send(...).end()`

File: src/app/b2c/postChangeEmail.js (hunk 1 above)
Comment (joseluisgraa) [reply src/app/b2c/postChangeEmail.js]: changed

File: src/app/b2c/postChangeEmail.js (hunk 2 above)
Comment (joseluisgraa) [reply src/app/b2c/postChangeEmail.js]: done

File: src/app/b2c/postChangeEmail.js (hunk 3 above)
Comment (joseluisgraa) [reply src/app/b2c/postChangeEmail.js]: done

**Pull request merged**